kernel-boot/modules/rdma.conf

```diff
@@ -0,0 +1,21 @@
+# These modules are loaded by the system if any RDMA devices is installed
+# iSCSI over RDMA client support
+ib_iser
+
+# iSCSI over RDMA target support
+# ib_isert
+
+# User access to RDMA verbs (supports libibverbs)
+ib_uverbs
+
+# User access to RDMA connection management (supports librdmacm)
+rdma_ucm
+
+# RDS over RDMA support
+# rds_rdma
```

Comment:
The rds_rdma module should be loaded only if rds is loaded.  What we need is a persistent systemd environment variable to signal to a separate rds systemd service that is should include rds_rdma.